### PR TITLE
Filter out duplicate and bad ROAs

### PR DESCRIPTION
**Pull request by taktv6**
This is functionality moving from our internal version to the public.

### CODE REVIEW

File: cmd/octorpki/filter.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package main
+
+import "github.com/cloudflare/gortr/prefixfile"
+
+func FilterBad(roalist []prefixfile.ROAJson) ([]prefixfile.ROAJson, error) {
```
Comment (BarbarossaTM) [inline cmd/octorpki/filter.go:5]: Can we rename this to `FilterInvalidPrefixLen` or something? :-)

Comment (taktv6) [reply cmd/octorpki/filter.go:5]: Not in this pr. As said in the description: Just moving existing code here.

Comment (taktv6) [reply cmd/octorpki/filter.go]: Well, okay. I took the time now to clean this up a bit.

Comment (BarbarossaTM) [reply cmd/octorpki/filter.go]: Grazie

**Pull request merged**